tasks/tasks.go

```diff
@@ -11,7 +11,24 @@ import (
 	"github.com/google/ts-bridge/stackdriver"
 	"github.com/google/ts-bridge/storage"
 	"github.com/google/ts-bridge/tsbridge"
+	"log"
 	"strings"
+	"sync"
+)
+
+var (
```

Comment:
readability:

convention is to have a single var declaration, instead of multiple groups.

```
var (
	sdClient     *stackdriver.Adapter
	sdClientOnce sync.Once

	metricConfig     *tsbridge.MetricConfig
	metricConfigOnce sync.Once

	statsCollector     *tsbridge.StatsCollector
	statsCollectorOnce sync.Once
)
```